frontend/app/src/App.test.tsx

```diff
@@ -532,6 +541,21 @@ describe("App", () => {
     const makeAppWithElements = async (): Promise<void> => {
       renderApp(getProps())
       sendForwardMessage("newSession", NEW_SESSION_JSON)
+      sendForwardMessage("navigation", {
+        ...NAVIGATION_JSON,
+        appPages: [
+          {
+            pageScriptHash: "page_script_hash",
+            pageName: "streamlit_app",
+            isDefault: true,
+          },
+          {
+            pageScriptHash: "other_page_script_hash",
+            pageName: "streamlit_app",
+            isDefault: true,
```

Comment:
curiosity question: can two pages have the `isDefault: true` field? Will the last one overwrite the first one?